src/validators/stats/index.js

```diff
@@ -1,18 +1,41 @@
-import {sortBy, difference, includes} from 'lodash'
+import {sortBy, difference} from 'lodash'
 import {friendlyTypeOf} from '../../utils'
+import _ from 'lodash'
+
+// available keys aren't really documented as far as I know,
+// but I got these by looking at the source:
+// https://github.com/webpack/webpack/blob/9f440e30ecac00bfc27b91d372a969d3414d194c/lib/Stats.js
+const validProperties = sortBy([
```

Comment:
I kind of liked putting all these below just to reduce what's on top, but *shrug*